src/DetailsView/components/failure-instance-panel-control.tsx

```diff
@@ -19,20 +20,24 @@ import { GenericPanel, GenericPanelProps } from './generic-panel';
 export interface FailureInstancePanelControlProps {
     step: string;
     test: VisualizationType;
-    addFailureInstance?: (description, test, step) => void;
-    editFailureInstance?: (description, test, step, id) => void;
+    addFailureInstance?: (instanceData, test, step) => void;
+    editFailureInstance?: (instanceData, test, step, id) => void;
     actionType: CapturedInstanceActionType;
+    originalInstance?: FailureInstanceData;
```

Comment:
I'm not sure if `originalInstance` is a good name here. Original implies there some context to how this property must be used for this control but I don't think that's the case.

A better name would be 'failureInstance' or even 'instance' since that implies no linking with any sort of context of how this control is used.

Reply:
I was thinking this parameter is only filled if we open the panel with the intention of "editing" so was trying to make it clear that the passed in instance was the one we were going to be editing. Do you think it's best to just assume it's clear when we use it to fill the "current instance" field?

Reply:
Nvm--discussed offline and changing to preserve the ambiguity of the panel!